### PR TITLE
feat([TD-12441]): support JSON type in tag, also fix TD-12292

### PR DESCRIPTION
**Pull request by zitsen**

### CODE REVIEW

File: src/stmt.rs [hunk 1]
```diff
@@ -185,33 +185,36 @@ mod test {
...
+    _test_column_null!(utinyint, "tinyint unsigned");
+    _test_column_null!(usmallint, "smallint unsigned");
+    _test_column_null!(uint, "int unsigned");
+    _test_column_null!(ubigint, "bigint unsigned");
+    _test_column_null!(timestamp, "timestamp");
+    _test_column_null!(float, "float");
+    _test_column_null!(double, "double");
+    _test_column_null!(binary, "binary(10)");
+    _test_column_null!(nchar, "nchar(10)");
     #[tokio::test]
```
Comment (sangshuduo) [inline src/stmt.rs:218]: no test for JSON?

**Review by sangshuduo — COMMENTED** [review]
no test for JSON

File: src/stmt.rs (hunk 1 above)
Comment (zitsen) [reply src/stmt.rs:218]: json only supported in tag. All test is in client::test::json_tag. STMT json test is in stmt::test::json .

**Review by sangshuduo — APPROVED** [review]
LGTM

**Pull request merged**